tests/test_build_oid_schema.py

```
@@ -0,0 +1,81 @@
+import pytest
+from unittest.mock import MagicMock, patch
```

Ruff (0.11.9)

`unittest.mock.patch` imported but unused

Remove unused import: `unittest.mock.patch`

(F401)